.github/workflows/test.yml

```
@@ -14,4 +14,7 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
       - name: Test
-        run: go test -race ./...
+        run: go test -race -coverprofile=coverage.txt -covermode=atomic ./...
```

```suggestion
        run: go test -race -coverprofile=coverage.txt -covermode=atomic -coverpkg=./... ./...
```
By default, go test will only count packages that it finds tests in. If you ever forget to put even just an empty `_test.go` file in a package, then it won't count for the coverage percentage. Specifying `-coverpkg=./...` fixes that and will use all packages for calculating coverage.

---

I'm trying to inflate the numbers come on.